/cita/modules/python/3.10.2/lib/python3.10/site-packages/rebound/simulationarchive.py:146: RuntimeWarning: You have to reset function pointers after creating a reb_simulation struct with a binary file.
  warnings.warn(message, RuntimeWarning)
Traceback (most recent call last):
  File "/fs/lustre/scratch/RAID-CITA/ksmith/runs/planet_hist.py", line 55, in <module>
    planet_hist(hist_sim_arch_directory, hist_xarch)
  File "/fs/lustre/scratch/RAID-CITA/ksmith/runs/planet_hist.py", line 46, in planet_hist
    a_final = sim[last_snapshot-1][0].particles[planet].a
  File "/cita/modules/python/3.10.2/lib/python3.10/site-packages/rebound/simulation.py", line 2307, in __getitem__
    raise AttributeError("Index {0} used to access particles out of range.".format(key))
AttributeError: Index 3 used to access particles out of range.
